exporter/redis.go

```diff
@@ -711,27 +713,34 @@ func scanForKeys(c redis.Conn, pattern string) ([]string, error) {
 			return keys, fmt.Errorf("invalid response from SCAN for pattern: %s", pattern)
 		}
 
-		k, _ := redis.Strings(arr[1], nil)
+		k, err := redis.Strings(arr[1], nil)
+		if err != nil {
```

Comment:
I don't think the is is necessary but if you really want to check for and  error then you should take appropriate action if there is one, e.g not append k to keys if err != nil.